packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1000,10 +1005,6 @@ export class ContainerRuntime
 		}
 
 		const createIdCompressorFn = async (): Promise<IIdCompressor & IIdCompressorCore> => {
```

Comment:
Should createIdCompressorFn  be removed as well / inlined? I think it's not really needed any more

Reply:
made is sync but lets keep the function for now as it will called more than once.